index.md

```diff
@@ -83,37 +62,58 @@ We weight grades as follows:
 | Final Project | 40% | 25% |  
 | 4th Credit Project | | 15% |  
 
-You can expect the cutoffs to be approximately as follows
-98 - 100: A+  
-93 - 98: A
-90 - 93: A-
-87 - 90: B+
-83 - 87: B
-80 - 83: B-
-77 - 80: C+
-73 - 77: C
-70 - 73: C-
-60 - 70: D
-Below 60: F
+You can expect the cutoffs to be approximately as follows: 
 
+- [90, 100]: A-, A, A+
```

Comment:
This line doesn't render for me, I think Markdown considers square brackets (`[ ... ]`) a link unless it's escaped.
```suggestion
- \[90, 100\]: A-, A, A+
```